ott/core/icnn.py

```diff
@@ -88,66 +43,142 @@ class ICNN(nn.Module):
       `jax.nn.initializers.normal`).
     act_fn: choice of activation function used in network architecture
       (needs to be convex, default: `nn.leaky_relu`).
+    pos_weights: choice to enforce positivity of weight or use regularizer.
+    gaussian_maps: data inputs of source and target measures for
+      initialization scheme based on Gaussian approximation of input and
+      target measure (if None, identity initialization is used).
   """
 
   dim_hidden: Sequence[int]
-  init_std: float = 0.1
+  init_std: float = 1e-1
   init_fn: Callable = jax.nn.initializers.normal
-  act_fn: Callable = nn.leaky_relu
+  act_fn: Callable = nn.relu
   pos_weights: bool = True
+  dim_data: int = 2
+  gaussian_map: Tuple = None
```

Comment:
same, maybe define what's in the Tuple ? it's also only 2 elements, right?

Reply:
Done!